City-of-Pori/pori-kada
# PORI-118: Heading font sizes

Lowered heading font-sizes a bit.

To test:
1. drush cc all
2. go edit any page or go to a page that has different sizes of heading
3. make sure that each heading from h1 to h5 look better in size

Reference images: [Before](https://monosnap.com/image/KNks1XlmyjZ9ogQPUmedm9K57FIMX0) ->[After](https://monosnap.com/image/nHsBxScigMfiJRfl7I3QEbzORsAhUT)
